public/app/core/services/FetchQueueWorker.ts

```diff
@@ -14,8 +14,7 @@ interface WorkerEntry {
 
 export class FetchQueueWorker {
   constructor(fetchQueue: FetchQueue, responseQueue: ResponseQueue, config: GrafanaBootConfig) {
-    const maxParallelRequests = config?.http2Enabled ? 1000 : 5; // for tests that don't mock GrafanaBootConfig the config param will be undefined
-
+    const maxParallelRequests = config?.http2Enabled ? 1000 : config.http1MaxParallelRequests; // for tests that don't mock GrafanaBootConfig the config param will be undefined
```

Comment:
Should use `config?.http1MaxParallelRequests` to be consistent with how http2 setting is used?